tests/gradle.properties

```diff
@@ -6,3 +6,7 @@ kotlin.mpp.stability.nowarn=true
 xcodeproj=samples/multiplatform/kmp-ios-app/kmp-ios-app.xcodeproj
 
 android.useAndroidX=true
+
+# Enable the build cache
+# See https://docs.gradle.org/current/userguide/build_cache.html
+org.gradle.caching=true
```

Comment:
Not sure this is needed or if the one in the root project is enough.

Reply:
It's technically an included build and not the root project so I would expect it to be required (but I'm not sure).

Reply:
Confirmed: it's needed.

```
./gradlew -p tests :enums:check
./gradlew -p tests :enums:clean
./gradlew -p tests :enums:check
```

- Without it, tests run again
- With it: `Task :enums:test FROM-CACHE`

Reply:
👍 . That being said, running tests from cache seems... a good way to avoid seeing things that break... But we'll see...